ios/Text/RNSVGTSpan.m

```diff
@@ -1061,24 +1030,18 @@ + (CGFloat)getTextAnchorOffset:(RNSVGTextAnchor)textAnchor width:(CGFloat) width
 
 - (void)setupTextPath:(CGContextRef)context
 {
-    lines = nil;
-    lengths = nil;
     textPath = nil;
     RNSVGText *parent = (RNSVGText*)[self superview];
-
     while (parent) {
         if ([parent class] == [RNSVGTextPath class]) {
             textPath = (RNSVGTextPath*) parent;
-            [textPath getPathLength:&_pathLength
-                          lineCount:&lineCount
-                            lengths:&lengths
-                              lines:&lines
-                           isClosed:&isClosed];
+            RNSVGNode *template = [self.svgView getDefinedTemplate:textPath.href];
+            CGPathRef path = [template getPath:nil];
+            [measure extractPathData:path];
```

Comment:
Need to reset measure if transitioning from having a textPath to not having one.